src/utils/strip-whitespace.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Remove sequences of whitespace greater than 1 character from `str`
+ *
+ * @param {string} str
+ * @returns {string}
+ */
+const strip = (str) => str.replace(/\s+/g, ' ');
```

Comment:
Well, I don't think it's correct logic to compare strings. You basically replace all continuous spaces to only one space, no matter where they are. It will make the following queries the same:

```
'select * from pizza where pizza_name = "Hello      Pizza"'
```

```
'select * from pizza where pizza_name = "Hello Pizza"'
```

Reply:
Found this package should probably help: https://www.npmjs.com/package/dedent

Reply:
That's a good point, although I haven't ever actually used string literals with spaces in this code. It seemed like a reasonable thing to do at the time since after trying `dedent` and still having issues with strings not matching. I will revisit using dedent again though to see if I can fix it.

Reply:
This is officially fixed and dedent is working.